README.md

```diff
@@ -1,5 +1,6 @@
-[![Code Climate](https://codeclimate.com/github/codeclimate/php-test-reporter.svg)](https://codeclimate.com/github/codeclimate/php-test-reporter)
 [![Build Status](https://travis-ci.org/codeclimate/php-test-reporter.svg?branch=master)](https://travis-ci.org/codeclimate/php-test-reporter)
+[![Code Climate](https://codeclimate.com/github/codeclimate/php-test-reporter.svg)](https://codeclimate.com/github/codeclimate/php-test-reporter)
+[![Test Coverage](https://codeclimate.com/github/codeclimate/php-test-reporter/badges/coverage.svg)](https://codeclimate.com/github/codeclimate/php-test-reporter/coverage)
```

Comment:
Seems like the coverage is currently not reported, see https://codeclimate.com/github/codeclimate/php-test-reporter:

![screen shot 2017-01-10 at 19 16 34](https://cloud.githubusercontent.com/assets/605483/21818769/59b83558-d769-11e6-8077-b53724cf01dd.png)

and https://travis-ci.org/codeclimate/php-test-reporter/builds/187500907#L256-L256:

```
$ php composer/bin/test-reporter

Unexpected response: 100 Continue
HTTP/1.1 401 Unauthorized
Date: Thu, 29 Dec 2016 16:43:24 GMT
Content-Type: text/plain; charset=utf-8
Transfer-Encoding: chunked
Connection: keep-alive
Status: 401 Unauthorized
Cache-Control: no-cache
X-XSS-Protection: 1; mode=block
X-Request-Id: a5a9731a-0941-4a76-9e79-03407ac1ba62
X-Revision: 6a629dc4ec6f6c7bd3c51176346ca3397ef27254
X-Runtime: 0.012891
X-Frame-Options: SAMEORIGIN
X-Content-Type-Options: nosniff
X-Powered-By: Phusion Passenger 5.0.22
Server: nginx + Phusion Passenger 5.0.22
Repo not found.
```